octavia_f5/controller/worker/flows/f5_flows.py

```diff
@@ -18,11 +18,16 @@
 
 from octavia.network import data_models as network_models
 from octavia_f5.controller.worker.tasks import f5_tasks
+from octavia_f5.utils import driver_utils
 
 LOG = logging.getLogger(__name__)
 
 
 class F5Flows(object):
+
+    def __init__(self, tasks=f5_tasks):
+        self.tasks = tasks.F5Tasks()
```

Comment:
Why did you decide to do it like this and put all task classes into one big class even for the old module? Why cannot we choose/pass the module that we need and keep naming for the tasks the same in both of them? In this case you do not need to change old  module (put everything to one big class) much and the code is much simplier for review.